projects.yaml

```diff
@@ -63,7 +63,9 @@ labels:
 projects:
 - name: Material for MkDocs
   mkdocs_theme: material
-  mkdocs_plugin: [material/search, material/social, material/tags]
+  mkdocs_plugin: [material/blog, material/group, material/offline, material/search, material/social, material/tags]
```

Comment:
If this should serve as a demo configuration: please don't add the offline plugin – it should only be enabled for when you want to serve from `file://`. Additionally, the blog is optional and will automatically create the files necessary when they aren't there, which is not what most users probably want.

Reply:
https://github.com/mkdocs/catalog#-theming

I added a separation there so it's not exactly like one full demo configuration

Reply:
Ah okay, great, let's try that ☺️